atomic

```diff
@@ -119,6 +116,15 @@ if __name__ == '__main__':
     parser.add_argument('--debug', default=False, action='store_true')
     subparser = parser.add_subparsers(help=_("commands"))
 
+    # atomic cluster (if available)
+    try:
+        import commctl.cli
+        clusterp = subparser.add_parser(
```

Comment:
Not crazy on this, since I think we should just require it.  Having it be optional makes us shipping the man page seem bad.  Remove the try block.